### PR TITLE
Add support for fragment configuration file

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by p2kmgcl** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: fixtures/sample-project-content/src/sample-collection/sample-component-fragment/fragment-configuration.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (p2kmgcl) [inline fixtures/sample-project-content/src/sample-collection/sample-component-fragment/fragment-configuration.json:1]: @victorg1991 can you add fixtures for projects with/without fragment-configuration?

File: fixtures/sample-project-content/src/sample-collection/sample-component-fragment/fragment.json [hunk 2]
```diff
@@ -3,5 +3,6 @@
   "htmlPath": "sample-fragment.html",
   "cssPath": "sample-fragment.css",
   "jsPath": "sample-fragment.js",
+  "fragmentConfigurationPath": "fragment-configuration.json",
```
Comment (p2kmgcl) [inline fixtures/sample-project-content/src/sample-collection/sample-component-fragment/fragment.json:6]: @victorg1991 please can you check if this configuration is being read? (Try placing config/html/css/js in different directories, etc etc)

**Review by p2kmgcl — CHANGES_REQUESTED** [review]

File: fixtures/sample-project-content/src/sample-collection/sample-component-fragment/fragment.json (hunk 2 above)
Comment (victorg1991) [reply fixtures/sample-project-content/src/sample-collection/sample-component-fragment/fragment.json:6]: yes

**Review by p2kmgcl — APPROVED** [review]

**Pull request merged**